client/src/partials/patient_invoice/patientInvoice.html

```diff
@@ -35,8 +35,9 @@
                   <select
                     class="form-control"
                     name="service"
-                    ng-model="PatientInvoiceCtrl.Invoice.details.service_id"
-                    ng-options="service.id as service.name for service in PatientInvoiceCtrl.services"
+                    ng-model="PatientInvoiceCtrl.Invoice.service"
```

Comment:
The end to end tests use the model 'service_id` to locate this component. They will need to be updated to pass with this model/ value change.